stdlib/number.gr

```diff
@@ -735,14 +741,18 @@ provide let linearMap = (inputRange, outputRange, current) => {
   if (isNaN(current)) {
     current
   } else {
-    if (isInfinite(inputRange.rangeStart) || isInfinite(inputRange.rangeEnd))
-      throw Exception.InvalidArgument("The inputRange must be finite")
-    if (isNaN(inputRange.rangeStart) || isNaN(inputRange.rangeEnd))
-      throw Exception.InvalidArgument("The inputRange must not include NaN")
-    if (isInfinite(outputRange.rangeStart) || isInfinite(outputRange.rangeEnd))
-      throw Exception.InvalidArgument("The outputRange must be finite")
-    if (isNaN(outputRange.rangeStart) || isNaN(outputRange.rangeEnd))
-      throw Exception.InvalidArgument("The outputRange must not include NaN")
+    if (isInfinite(inputRange.rangeStart) || isInfinite(inputRange.rangeEnd)) throw Exception.InvalidArgument(
```

Comment:
I think the old behaviour was better here too. where it prioritized putting it onto a newline before breaking.

Reply:
Good catch, that was just a mistake. Fixed.